Fix tests for 32bit builds until we can bring these upstream
diff --git a/kmymoney/mymoney/storage/mymoneyseqaccessmgrtest.cpp b/kmymoney/mymoney/storage/mymoneyseqaccessmgrtest.cpp
index dcb4b4a..e803203 100644
--- a/kmymoney/mymoney/storage/mymoneyseqaccessmgrtest.cpp
+++ b/kmymoney/mymoney/storage/mymoneyseqaccessmgrtest.cpp
@@ -58,13 +58,13 @@ void MyMoneySeqAccessMgrTest::testEmptyConstructor()
   QCOMPARE(m->m_nextPayeeID, 0ul);
   QCOMPARE(m->m_nextScheduleID, 0ul);
   QCOMPARE(m->m_nextReportID, 0ul);
-  QCOMPARE(m->m_institutionList.count(), 0ul);
-  QCOMPARE(m->m_accountList.count(), 5ul);
-  QCOMPARE(m->m_transactionList.count(), 0ul);
-  QCOMPARE(m->m_transactionKeys.count(), 0ul);
-  QCOMPARE(m->m_payeeList.count(), 0ul);
-  QCOMPARE(m->m_tagList.count(), 0ul);
-  QCOMPARE(m->m_scheduleList.count(), 0ul);
+  QCOMPARE(m->m_institutionList.count(), (size_t)0);
+  QCOMPARE(m->m_accountList.count(), (size_t)5);
+  QCOMPARE(m->m_transactionList.count(), (size_t)0);
+  QCOMPARE(m->m_transactionKeys.count(), (size_t)0);
+  QCOMPARE(m->m_payeeList.count(), (size_t)0);
+  QCOMPARE(m->m_tagList.count(), (size_t)0);
+  QCOMPARE(m->m_scheduleList.count(), (size_t)0);
 
   QCOMPARE(m->m_dirty, false);
   QCOMPARE(m->m_creationDate, QDate::currentDate());
